packages/schema-utils/tests/__snapshots__/codegen.input.ts

```diff
@@ -0,0 +1,2611 @@
+// This file is auto generated from data/messages/message.json
```

Comment:
Yeah, this test basically takes the entire `messages.ts` and compares it to a snapshot. It was more for development purposes because I was interested in `messages.ts` in particular